src/v0/destinations/google_adwords_offline_conversions/utils.test.js

```diff
@@ -273,3 +274,97 @@ describe('getClickConversionPayloadAndEndpoint util tests', () => {
     );
   });
 });
+
+describe('populateConsentForGoogleDestinations', () => {
```

Comment:
Please add a case where only partial consents data is available.

Reply:
added